src/Lykke.Service.MarketMakerReports.Core/Domain/InventorySnapshots/AssetBalanceInventory.cs

```diff
@@ -9,5 +11,20 @@ public class AssetBalanceInventory
         public IReadOnlyList<AssetInventory> Inventories { get; set; }
         
         public IReadOnlyList<AssetBalance> Balances { get; set; }
+
+        public AssetBalance GetBalanceForExchange(string exchange)
```

Comment:
GetBalanceByExchange?

Reply:
Renamed. Maybe GetBalanceOnExchange is another better option.